DemoApp/ChatPresenter.swift

```diff
@@ -199,18 +215,54 @@ class DemoChannelListVC: ChatChannelListVC {
     /// The `UIButton` instance used for navigating to new channel screen creation,
     lazy var createChannelButton: UIButton = {
         let button = UIButton()
-        button.setImage(UIImage(named: "pencil")!, for: .normal)
+        button.setImage(UIImage(systemName: "plus.message")!, for: .normal)
+        return button
+    }()
+    
+    lazy var hiddenChannelsButton: UIButton = {
+        let button = UIButton()
+        button.setImage(UIImage(systemName: "archivebox")!, for: .normal)
```

Comment:
I know this is the sample app, but should we still safely unwrap?

Reply:
I think it is fine, besides being the sample app, in the case that the image doesn't exist, or apple changes the name of the icon, the crash will help us update it since it is a crash that happens almost at app launch 👍